sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -965,3 +966,14 @@ executing the same control flow (without introducing significant overhead). If
 we decide at a later date that `tangle_group` should support only sub-groups,
 we should revisit the name to avoid creating confusion.
 --
+
+. Inconsistencies in `fixed_size_group`
++
+--
+`fixed_size_group` accepts template argument `PartitionSize` of type
+`std::size_t`, which can be an unsigned 64-bit integer. However,
+`linear_id_type` is defined as `uint32_t` for this group. Even though with the
+current restrictions it is impossible to encounter such situation, there is
+still a mismatch allowing to create a group so big that linear IDs won't be
+correct for some of its work-items.
+--
```

Comment:
I think this is just a bug, and we should update `linear_id_type` with `size_t`.

My initial prototype only works with sub-groups (which are limited to `uint32_t`) and that's probably how this crept in.  If/when we allow partitioning of work-groups (and other groups) into fixed-size groups, we may need `size_t`.